### DISCUSSION TITLE
Support for Asciidoc-like code callouts

### DISCUSSION THREAD

**Discussion by dodalovicgran** [facebook/docusaurus#8751] [Q&A]
Hi, 
I was wondering if there's such possibility with Docusaurus?

![image](https://user-images.githubusercontent.com/116893230/223946729-cf28bbb5-d168-4e03-85b6-de3a7358c3ab.png)

**Comment by slorber** [comment]
Not sure it's possible currently but maybe with code magic comments?
https://docusaurus.io/docs/markdown-features/code-blocks#custom-magic-comments

If it's not possible we could enable magic comments to have extra metadata.

What's the asciidoc syntax to achieve the above result?

**Comment by dodalovicgran** [comment]
From [source](https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/)

![image](https://user-images.githubusercontent.com/116893230/223987655-b5d8f452-3d03-451f-9f64-5d0d1e920f03.png)

**Reply by slorber** [reply]
Thanks

That seems achievable to implement in MDX (maybe some others already built this?) but maybe not with the exact same syntax `<1>`

**Reply by dodalovicgran** [reply]
It's any kind of comment, so most often `//1`, `#1` will do the trick as well. Dependent on the comment feature of the language
